great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -285,26 +306,22 @@ def get_expect_values_to_be_between_chart(
             alt.Chart(data=df)
             .mark_area()
             .encode(
-                x=alt.X(
-                    domain_name,
-                    type=domain_type,
-                    title=domain_title,
-                ),
-                y=alt.Y(min_value, title=metric_title, type=metric_type),
-                y2=alt.Y2(max_value, title=metric_title),
+                x=domain_component.plot_on_axis(),
+                y=metric_component.plot_on_axis(shorthand=min_value_component.name),
+                y2=alt.Y2(max_value_component.name, title=metric_component.title),
```

Comment:
Shouldn't this be `field=min_value_component.name`? I'm struggling to follow the logic here. I feel like we should have a sub-class called `ExpectationKwargPlotComponent` that is used here for `min_value` and `max_value`.